### DISCUSSION TITLE
AOSP 12.1 - arm32_binder64-ab- (FLOSS)

### DISCUSSION THREAD

**Discussion by GSIPRO** [phhusson/treble_experimentations#2343] [Ideas]
Monsieur, 
is there a realistic chance that you will build an AOSP 12.1 GSI arm32_binder64-ab-FLOSS (with microG) in the near future? 
If yes, when? 
If not, why not? 
I like to hear from you. 
Sulue.

**Comment by GSIPRO** [comment]
No answer is also an answer, but not to be surpassed in arrogance.

**Comment by phhusson** [comment]
Flods arm32_binder64 was initially removed because it had Firefox and Firefox doesn't work on 32bits. I've since then moved to Bromite so I'll check.

30 mai 2022 00:12:44 GSIPRO ***@***.***>:
